src/CmapProcessor.js

```diff
@@ -156,15 +156,21 @@ export default class CmapProcessor {
 
     let selectors = this.uvs.varSelectors.toArray();
     let i = binarySearch(selectors, x => variationSelector - x.varSelector);
+    if (i === -1) {
+      return 0;
+    }
     let sel = selectors[i];
 
-    if (i !== -1 && sel.defaultUVS) {
+    if (sel.defaultUVS) {
       i = binarySearch(sel.defaultUVS, x =>
         codepoint < x.startUnicodeValue ? -1 : codepoint > x.startUnicodeValue + x.additionalCount ? +1 : 0
       );
+      if (i !== -1) {
```

Comment:
Should this be the opposite, i.e. `i === -1`?

Reply:
I'm sorry I didn't make it clear enough. My pull request fixes two problems.

1) When the default UVS **table** exists and its **entry** not exists (`i === -1`), the variable `i` was overwriten and the non-default UVS isn't used.
2) When the default **entry** exists ( `i !== -1` ), it means "the default glyph (the same glyph as base character) should be used" and checking the non-default UVS is unnecessary. By returning 0, the Variation Selector is ignored correctly here.

Reply:
A test would help understanding the issue and ensuring the fix is correct